Assets/StreamingAssets/Text/Master Localization CSV Files/Internal_Strings.csv

```diff
@@ -1,67 +1,66 @@
-lockpickingSuccess,The lock clicks open.
-lockpickingFailure,It does not unlock.
-magicLock,This is a magically held lock...
-lockpickChance1,This lock has nothing to fear from you...
-lockpickChance2,It'd be a miracle if you picked this lock...
-lockpickChance3,This lock looks to be beyond your skills...
+Key,Value,,,
+about_internal_strings,This string table collection stores text that was hard-coded in FALL.EXE and text unique to Daggerfall Unity.,,,
+pleaseSelectYourHomeProvince,Please select your home province...,,,
+saveGame,Save Game,,,
+loadGame,Load Game,,,
+thingJustDied,%s just died.,,,
+gameSaved,Game saved.,,,
+gameLoaded,Game loaded.,,,
+thisHouseHasNothingOfValue,This house has nothing of value.,,,
+findLocationPrompt,Enter name of place : ,,,
+rightHandEquipped,Right hand equipped.,,,
+leftHandEquipped,Left hand equipped.,,,
+usingRightHand,Using weapon in right hand.,,,
+usingLeftHand,Using weapon in left hand.,,,
+itemHasBroken,%s has broken.,,,
+itemHasBrokenPlural,%s have broken.,,,
+cannotCarryGold,You cannot carry that much gold.,,,
+cannotFloat,You are carrying too much to stay afloat.,,,
+youHaveNoArrows,You have no arrows.,,,
+oldSaveNoTrade,"Old indoor save loaded, trade window will not work please exit and re-enter building.",,,
+theBodyHasNoTreasure,The body has no treasure.,,,
+youAreTooFarAway,You are too far away...,,,
+youSeeAn,You see an %s.,,,
+youSeeA,You see a %s.,,,
+youSeeADeadPerson,You see a dead person.,,,
+youSeeADead,You see a dead %s.,,,
+youCollectArrows,You pluck your arrows out of the corpse.,,,
+loiterHowManyHours,Loiter how many hours : ,,,
+restHowManyHours,Rest how many hours : ,,,
+potionRecipeFor,Recipe for Potion of %po,,,
+potionOf,Potion of %po,,,
+letterPrefix,Letter: ,,,
+multipleAssignments,You have multiple assignments...,,,
+youAreEntering,You are entering %s,,,
+storeClosed,Store is closed. Open from %d1:00 to %d2:00.,,,
+guildClosed,Guild is closed. Open from %d1:00 to %d2:00.,,,
+any,Any %s,,,
+arkayDesc,God of Birth and Death,,,
+zenDesc,God of Work and Commerce,,,
+maraDesc,Goddess of Love,,,
+akatoshDesc,God of Time,,,
+julianosDesc,God of Logic,,,
+dibellaDesc,Goddess of Beauty,,,
+stendarDesc,God of Mercy,,,
+kynarethDesc,Goddess of Air,,,
+interactionIsNowInMode,Interaction is now in %s mode.,,,
+steal,steal,,,
+grab,grab,,,
+info,info,,,
+dialogue,dialogue,,,
+residence,Residence,,,
+youSee,You see %s.,,,
+theNamedResidence,The %s Residence,,,
+playerResidence,%s's house,,,
+materialIneffective,The material of the weapon you are using is ineffective.,,,
+successfulBackstab,Successful backstab!,,,
+languagePacified,Pacified %e using %s skill.,,,
+lockpickingSuccess,The lock clicks open.,,,
+lockpickingFailure,It does not unlock.,,,
+magicLock,This is a magically held lock...,,,
+lockpickChance1,This lock has nothing to fear from you...,,,
+lockpickChance2,It'd be a miracle if you picked this lock...,,,
+lockpickChance3,This lock looks to be beyond your skills...,,,

```

Comment:
This file has extra columns. Not an error by itself and the file will still work, but let's clean it

Reply:
Turns out the semicolons in row 626 caused the data to get split into new cell columns by LibreOffice Calc, since by default it splits by semicolon. The other CSV files either didn't have any semicolons, or they were all contained within double quotes, preventing them from causing a split.
It seems I was using an outdated internal_strings.csv, but updating didn't completely fix things, either. Eventually I loaded up the stock internal_strings.csv in a hex editor and it seems to have inconsistent formatting; some linebreaks within cells use just $0A, while others use $0D0A. I only had a few changes in that file so I just used the hex editor to make the changes in the end, not touching the weird formatting at all. Hopefully I can figure out how to update this PR.

Reply:
I think I merged it backwards somehow? Gonna delete this tomorrow and try again, sigh.